flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -901,10 +885,6 @@ private ApplicationReport startAppMaster(
 			}
 		}
 
-		final Path yarnFilesDir = getYarnFilesDir(appId);
```

Comment:
Why don't we need the permission setting anymore?

Reply:
It is happening in the `YarnApplicationFileUploader.getApplicationDir()`.